Bugzilla/Template.pm

```diff
@@ -195,12 +196,11 @@ sub renderComment {
 
     # Provide tooltips for full bug links (Bug 74355)
     my $urlbase_re = '(' . quotemeta(Bugzilla->localconfig->{urlbase}) . ')';
-    $text =~ s~\b(${urlbase_re}\Qshow_bug.cgi?id=\E([0-9]+)(\#c([0-9]+))?)\b
+    $text =~ s~(?<!\]\()\b(${urlbase_re}\Qshow_bug.cgi?id=\E([0-9]+)(\#c([0-9]+))?)\b
```

Comment:
Here we check that this full bug link is not being used in a markdown link [ ]( ) tag. If it is being used in one, we leave it alone, the markdown parser will use the full link to generate the corresponding anchor tag.

With the other reorder in this PR and _without_ this change, an anchor tag would be rendered with invalid unicode characters (because the markdown parser will use the `"\x{FDD2}" . ($count-1) . "\x{FDD3}"` as part of the link but it outputs invalid unicode that does not match in line 280 of this function where we replace it with the correct link)